pywbem/_server.py

```diff
@@ -762,6 +764,34 @@ def get_central_instances(self, profile_path, central_class=None,
             implementation supports only the scoping class methodology.
             `None` will cause the scoping class methodology not to be attempted.
 
+          reference_direction (:term:`string`):
+            Defines the usage of the ResultRole parameter in the association
+            request that determines the relation between component registered
+            profiles.  The possible values are 'dmtf' and 'snia' the default is
+            'dmtf'. This parameter exists because of issues between DMTF and
+            SNIA in the the definition of CIM_ReferencedProfile and its
+            subclasses.
+
+            Generally the DMTF definition documented in DSP1033 and the
+            DMTF PRP (DSP1001) is that:
+
+            * Antecedent = referenced profile = component profile
+            * Dependent = referencing profile = autonomous profile
+
+            At the same time, for historical reasons, the SNIA SMI specification
+            is ambiguous in that the definition varies by document.
```

Comment:
The SMI-S specs have only a minor inconsistency between the various pieces, which is that Figure 56 in the SMI-S PRP uses Antecedent / Dependent opposite to all other places. That can well be considered an error in an example diagram, i.e. in a non-normative part of the spec. Otherwise, everything else is consistent in itself, but opposite to DMTF.
Bottom line, I would not say it is "ambiguous", or "varies by document", but instead simply that the definition is different from DMTF. If you want, you can add "except for a non-normative diagram", but I would not mention that.

Reply:
DONE